metadata.rb

```diff
@@ -12,3 +12,4 @@
 supports 'windows'
 
 depends 'windows', '~> 1.38'
+depends 'compat_resource'
```

Comment:
At this point I'd just remove this and set the minimum version of chef to 12.7 in the cookbook. It's current in 11-land which is entirely unsupported now. 12 even goes EOL come April.